pkg/agent/baker.go

```diff
@@ -807,5 +807,18 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"GetOutboundCommand": func() string {
 			return getOutBoundCmd(config, config.CloudSpecConfig)
 		},
+		"ShouldConfigureCustomCATrust": func() bool {
+			return config.CustomCATrustConfig != nil && len(config.CustomCATrustConfig.CustomCATrustCerts) > 0
+		},
+		"GetCustomCATrustConfigCerts": func() []string {
+			if config.CustomCATrustConfig != nil && config.CustomCATrustConfig.CustomCATrustCerts != nil {
```

Comment:
nit: if we didn't have a struct and only an array, could range directly. but maybe worth it for future changes...

Reply:
Im open to changing it to an array, at first I wanted to add additional stuff like timestamp in the struct, but decided to go with a prefix of zeros instead. What would you prefer?